### PR TITLE
Campaign finance

### LINKED ISSUE #1193: Campaign finance analytics
Query to figure out who is giving money to whom, example code (this code sample is not ksql-ready, it needs to be adapted):

```
SELECT affiliation, SUM(amount) AS amount
FROM (
  SELECT *
  FROM (
    SELECT
      t.amt AS amount,
      t.occupation AS occupation,
      c.affiliation AS affiliation,
    FROM (
      SELECT
        trans.TRANSACTION_AMT AS amt,
        trans.OCCUPATION AS occupation,
        cmte.CAND_ID AS CAND_ID
      FROM [campaign_funding.transactions] trans
      RIGHT OUTER JOIN EACH (
        SELECT
          CMTE_ID,
          FIRST(CAND_ID) AS CAND_ID
        FROM [campaign_funding.committees]
        GROUP EACH BY CMTE_ID ) cmte
      ON trans.CMTE_ID = cmte.CMTE_ID) AS t
    RIGHT OUTER JOIN EACH (
      SELECT
        CAND_ID,
        FIRST(CAND_PTY_AFFILIATION) AS affiliation,
      FROM [campaign_funding.candidates]
      GROUP EACH BY CAND_ID) c
    ON t.CAND_ID = c.CAND_ID )
  WHERE occupation CONTAINS "ENGINEER")
GROUP BY affiliation
ORDER BY amount DESC
```

The mechanics of matching can be similar to https://developer.confluent.io/tutorials/online-dating/confluent.html

Inspiration: https://kiosk-dot-codelabs-site.appspot.com/codelabs/cloud-bq-campaign-finance/index.html?index=..%2F..index#7

### PR DESCRIPTION
**Pull request by ybyzek**
### Description

#1193 

Sister PR: https://github.com/confluentinc/kafka-connect-datagen/pull/103 (anticipating future datagen usage)

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/campaign_finance/campaign-finance/confluent.html

### New tutorial checklist

- [x] Implement good test cases
- [x] Validate hyperlinks
- [x] Spell check (e.g. using `aspell`)
- [x] Full code validated in Confluent Cloud
- [x] SQL style/syntax consistent to existing recipes
- [x] Validate presentation with `bundle exec jekyll serve --livereload`
- [x] Tutorial added to appropriate `index.html` or `use-case.html` file
- [x] Manually validated with the new datagen https://github.com/confluentinc/kafka-connect-datagen/pull/103

### CODE REVIEW

**Comment by ybyzek** [comment]
Campaign finance test case passes

File: _includes/shared/markup/ccloud/manual_insert.adoc
Comment (bbejeck) [inline _includes/shared/markup/ccloud/manual_insert.adoc:1]: FYC: It's a minor nit, feel free to ignore
```suggestion
If you cannot connect to a real data source with properly formatted data, or if you just want to execute this tutorial without external dependencies, no worries! Remove the `CREATE SOURCE CONNECTOR` commands and link:#test-with-mock-data[insert mock data] into the streams.
```

**Review by bbejeck — APPROVED** [review]
One minor comment, but this LGTM

**Comment by ybyzek** [comment]
Thanks for the review @bbejeck !  I am waiting on a copy review from @sophiajiangg and then will merge

**Pull request merged**
